cookbook/integrations/flytekit_plugins/duckdb_examples/example.py

```diff
@@ -0,0 +1,157 @@
+"""
+DuckDB Example
+--------------
+
+In this example, we will see how to run SQL queries using DuckDB. 
+
+First, import the necessary libraries.
+"""
+
+import json
+from typing import List
+
+import pandas as pd
+import pyarrow as pa
+from flytekit import kwtypes, task, workflow
+from flytekit.types.structured.structured_dataset import StructuredDataset
+from flytekitplugins.duckdb import DuckDBQuery
+from typing_extensions import Annotated
+
+# %%
+# A simple ``SELECT`` query
+# =========================
+#
+# To run a simple ``SELECT`` query, initialize a ``DuckDBQuery`` task.
+simple_duckdb_query = DuckDBQuery(
+    name="simple_task",
+    query="SELECT SUM(a) FROM mydf",
+    inputs=kwtypes(mydf=pd.DataFrame),
+)
```

Comment:
is the only output type of this task a `StructuredDataset`? If so we should call that out explicitly

Reply:
Yeah, it's StructuredDataset always. Will add a note.